content/references/protocol/transactions/types/diddelete.md

```diff
@@ -0,0 +1,33 @@
+---
+html: diddelete.html
+parent: transaction-types.html
+blurb: Delete a DID.
+labels:
+  - DID
+status: not_enabled
+---
+# DIDDelete
```

Comment:
Should this note that there are no fields?

Reply:
Good idea, note added.